packages/kotti-ui/source/kotti-button/types.ts

```diff
@@ -0,0 +1,28 @@
+import { Yoco } from '@3yourmind/yoco'
+
+export namespace KottiButton {
+	export enum Type {
+		PRIMARY = 'primary',
+		SECONDARY = 'secondary',
+		TEXT = 'text',
+		DANGER = 'danger',
+	}
+
+	export enum Size {
+		SMALL = 'small',
+		MEDIUM = 'medium',
+		LARGE = 'large',
```

Comment:
I think this is a breaking change

Reply:
Can you explain what about this is a breaking change? I suppose you think that originally you would pass `"lg"` instead of `"large"` , mainly because @carsoli stated the same in a previous comment. The [documentation](https://3yourmind.github.io/kotti/usage/components/button) wouldn't suggest so and I think I'm  actually fixing a bug here because large buttons somehow were dropped. As you can see in the documentation the medium button and the large button have the same height.